### PR TITLE
Update User_Functions.c

### PR DESCRIPTION
**Pull request by FischerTG**
UwU senpai notice me. UwU pwease accewpt

### CODE REVIEW

**Comment by FischerTG** [comment]
It worky me happy uwu.
Senpai uwu
yubi yubi
a.

File: Template/User_Functions.c [hunk 1]
```diff
@@ -35,3 +35,21 @@ void ReadPort(int *iStatusArray)      //function for reading a full port into po
     iStatusArray[iCounter] = (P0 & (0x01 << iCounter)) >> iCounter; //reading port, masking and shiffting
   }
 }
+
+/*-----Function for blinking LED-----*/
+int BlinkLED(int iOn, int iOff, int* iTimer)	//function for LED blinking
+{
+	if(*iTimer < iOn)							//check if timer reached value of iOn
```
Comment (BakxY) [inline Template/User_Functions.c:42]: Better version:
*iTimer =< iOn
Please change uwu

File: Template/User_Functions.c [hunk 2]
```diff
@@ -35,3 +35,21 @@ void ReadPort(int *iStatusArray)      //function for reading a full port into po
...
+	{
+		return(1);								//return status 1
+	}
+	else if(*iTimer < (iOn + iOff))				//check if timer reached value of iOn + iOff
```
Comment (BakxY) [inline Template/User_Functions.c:46]: Better version:
*iTimer =< (iOn + iOff)
Please change uwu

**Review by BakxY — CHANGES_REQUESTED** [review]

File: Template/User_Functions.c (hunk 2 above)
Comment (FischerTG) [reply Template/User_Functions.c:46]: uwu it cowwect

**Review by BakxY — APPROVED** [review]
i have noticed you uwu

**Pull request merged**